api/translate/translate/v1/translate.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3";
+package translate.v1;
+
```

Comment:
From: https://developers.google.com/protocol-buffers/docs/reference/go-generated
In order to generate Go code, the Go package's import path must be provided for every .proto file (including those transitively depended upon by the .proto files being generated). There are two ways to specify the Go import path:

by declaring it within the .proto file, or
by declaring it on the command line when invoking protoc.
We recommend declaring it within the .proto file so that the Go packages for .proto files can be centrally identified with the .proto files themselves and to simplify the set of flags passed when invoking protoc. If the Go import path for a given .proto file is provided by both the .proto file itself and on the command line, then the latter takes precedence over the former.
The Go import path is locally specified in a .proto file by declaring a go_package option with the full import path of the Go package.

Reply:
Defined here
https://github.com/expect-digital/translate/blob/fbba7cbcb2094b848695ff61e9823652b1246aa8/api/translate/buf.gen.yaml#L4-L5

https://docs.buf.build/tour/use-managed-mode